README.md

```diff
@@ -160,10 +160,10 @@ Please note that `.calendarFor` and `.liturgicalDayFor` are **async**. For furth
 
 This 2 methods produces an `Array` of `LiturgicalDay` objects (by default, one object per each day of the year):
 
-```json5
+```ts
```

Comment:
Please keep `json5` here.
`ts` isn't valid in this situation, and we are also loosing some of the highlight colors in GitHub.

Reply:
I could not find difference between `json5` and `ts` syntax hightlighting on GitHub.com (web); I tested it before pushing this commit.

However, I keep it the way you wish. :smiley: